### PR TITLE
Delete user and all their articles

### PR DESCRIPTION
**Pull request by LeeRenJie**
Depends on #36

### CODE REVIEW

File: alpha-blog/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -35,6 +38,13 @@ def update
     end
   end
 
+  def destroy
+    @user.destroy
```
Comment (tlkiong) [inline alpha-blog/app/controllers/users_controller.rb:42]: Add a `!` here. `@user.destroy!`
This is cause if there is any error which prevents this from being destroyed, we will know

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: alpha-blog/app/controllers/users_controller.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ def create
     if @user.save
       session[:user_id] = @user.id
       flash[:notice] = "Welcome to Alpha Blog #{@user.username}, you have successfully signed up!"
-      redirect_to articles_path
+      redirect_to root_path
```
Comment (tlkiong) [inline alpha-blog/app/controllers/users_controller.rb:23]: Why this change in this PR? 
This PR says delete users. This is create o

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: alpha-blog/app/controllers/users_controller.rb (hunk 2 above)
Comment (LeeRenJie) [reply alpha-blog/app/controllers/users_controller.rb:23]: ooo wanted to change to another path😅

**Review by tlkiong — APPROVED** [review]
